content/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-root-acls.config

```diff
@@ -12,7 +12,7 @@ end
 
 create service user acs-commons-on-deploy-scripts-service with path system/acs-commons
 set ACL for acs-commons-on-deploy-scripts-service
-    allow jcr:read on /
+    allow jcr:versionManagement,jcr:read,rep:write,jcr:lockManagement,crx:replicate on /
```

Comment:
This is a lot, if there is some vulnerability with that service user one can do a lot of damage. I would rather restrict it more by default. Those permissions can be extended on a per use case basis if really necessary